src/basket/BasketMiniView.tsx

```diff
@@ -54,13 +58,34 @@ type BasketMiniViewTabProps = {
   closePanel: () => void;
 };
 
+export const updateResultsWithAccessionSubsets = (
+  results: APIModel[],
+  namespace: Namespace,
+  accessions: string[]
+) => {
+  return results.map((r, index) => {
+    if (namespace === Namespace.uniprotkb) {
+      const entry: UniProtkbAPIModel = { ...r };
```

Comment:
There is a Type error I am unable to solve. Please could you look at it?

Reply:
In this case you need to cast the object to the right model in order to narrow down from the too generic APIModel to the correct UniProtkbAPIModel.
You could have narrowed down the type by checking if the object contained fields specific to UniProtKB, or here since you know and check that the namespace is uniprotkb then I'd say it's OK to cast the object.

Reply:
I get it, thank you.